### ISSUE TITLE
Multiple issue tempaltes

### ISSUE THREAD

**Issue by freaktechnik** [mozillach/gh-projects-content-queue#102]
A repository can have multiple issue templates now: https://github.com/blog/2495-multiple-issue-and-pull-request-templates

There should be one for a normal tweet (the default template) and one for a retweet at least.

There should be links in the README to create a new issue of either type.

Not giving this easy bug, since it kind of depends on #75
